crates/rb-sys/Cargo.toml

```diff
@@ -2,19 +2,20 @@
 name = "rb-sys"
 version = "0.8.1"
 edition = "2018"
-links = "ruby"
 readme = "readme.md"
 categories = ["ruby", "external-ffi-bindings"]
 description = "Rust bindings for the CRuby API"
 license = "MIT OR Apache-2.0"
 repository = "https://github.com/oxidize-rb/rb-sys"
 homepage = "https://github.com/oxidize-rb/rb-sys"
 documentation = "https://docs.rs/rb-sys"
+links = "rb" # we name it like this so users can still use magnus / rutie
```

Comment:
I've removed the `links = "ruby"` from Magnus and added a feature to use `rb-sys`, plus a couple of functions to make it easier to work with the two libraries, the changes are here: https://github.com/matsadler/magnus/commit/238487c3b08710b979ab37604e5c5e224af13b5a. I'm hoping to get a release out in the next week or so.